src/Homapage.js

```diff
@@ -0,0 +1,153 @@
+  display: flex;
+  justify-content: center;
+`;
+
+const AllPinsWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  border: 1px solid blue;
+  height: auto;
+`;
+
+const PinWrapper = styled.div`
+  width: 325px;
+  border: 1px solid red;
+  border-radius: 20px;
+  margin: 10px;
+  background-color: lightblue;
+  position: relative;
+`;
+
+const PinImage = styled.img`
+  width: 325px;
+  border-radius: 20px;
+  cursor: pointer;
+`;
+
+const SaveButton = styled.button`
+  color: inherit;
+  border: none;
+  border-radius: 30px;
+  padding: 10px;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+  position: absolute;
+  right: 15px;
+  top: 15px;
+  font-size: 1.5rem;
+`;
+
+function Homapage() {
+
+    const [displayAddPin, setDisplayAddPin] = useState(false);
+
+    const showAddPin = ()=>{
+        setDisplayAddPin(true);
+    }
+
+  return (
+    <MainWrapper>
+      <AllPinsWrapper>
+
+        <PinWrapper>
+          <PinImage src={chicken} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
+          <PinImage src={flower} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
+          <PinImage src={kitty} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
+          <PinImage src={wolf} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
+          <PinImage src={bear} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
+          <PinImage src={frog} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
+          <PinImage src={death} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
+          <PinImage src={uglyflower} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
+          <PinImage src={trex} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
+          <PinImage src={frog} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
+          <PinImage src={duck} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
+          <PinImage src={cherry} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
+          <PinImage src={guitarfrog} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
+          <PinImage src={hands} />
+          <SaveButton onClick={showAddPin} >save</SaveButton>
+        </PinWrapper>
+        <PinWrapper>
```

Comment:
DRY(don't repeat yourself)
Put the image source into an array and use loop like map, forEach to render them.